ai/overview.md

```diff
@@ -36,15 +36,33 @@ To use the Telerik MAUI AI Coding Assistant, you need:
 @[template](/_contentTemplates/common/ai-coding-assistant.md#getting-started)
 * @[template](/_contentTemplates/common/ai-coding-assistant.md#number-of-requests)
 
-## Number of Requests
+## Recommendations
 
-The Telerik MAUI AI Coding Assistant allows the following maximum number of requests based on your [Telerik license type](https://www.telerik.com/purchase/faq/licensing-purchasing):
+Consider the following recommendations when working with the AI Coding Assistant:
 
-| License Type | Request Limit |
-|--------------|---------------|
-| Perpetual | 50 requests per year |
-| Subscription | Virtually unlimited with fair use threshold of 300 requests per day |
-| Trial | 300 requests per trial period (does not reset with new trial activations) |
+* Add NuGet packages/referenced assemblies for the Telerik UI for WPF product.
+* Set the .sln as a context.
+* When switching between tasks and files, start a new session in a new chat window to avoid polluting the context with irrelevant or outdated information.
+* At the time of publishing, Claude Sonnet 4 produces optimal results.
+
+## Usage Limits
+
+Access to the AI Coding Assistant depends on your [Telerik license type](https://www.telerik.com/purchase/faq/licensing-purchasing):
+
+Subscription License:
```

Comment:
Personally, I would apply some formatting to Subscription License, Perpetual License, etc. Either bold, `>caption`, or H3/H4